src/uct/ib/base/ib_iface.h

```diff
@@ -342,6 +342,9 @@ struct uct_ib_iface {
         enum ibv_mtu                 path_mtu;
         uint8_t                      counter_set_id;
         uct_ib_iface_send_overhead_t send_overhead;
+        uint8_t                      rcx_multi_path;
+        uint8_t                      dc_multi_path;
```

Comment:
why does the common struct have per-transport field?
I'd expect something like:
ucs_ternary_auto_value_t multi_path;

Reply:
fixed